### ISSUE TITLE
Ubuntu 16.04 can't open executable

### ISSUE THREAD

**Issue by vasikxl** [Jessecar96/SteamDesktopAuthenticator#448]
Hello,
I wanted to use SDA on my Ubuntu, but if I follow steps in readme, it doesn't work.
When I use mono `"/home/.../SteamDesktopAuthenticator/Steam Desktop Authenticator/bin/release/Steam Desktop Authenticator.exe"` (yeah, the path is right, it doesn't work even if I open it without using terminal), it returns this error: https://pastebin.com/vdhYNyfP
And when i build it in MonoDevelop, it returns one warning as well (don't know if it's relevant, but just in case it was): `/app/lib/mono/msbuild/15.0/bin/Microsoft.CSharp.CurrentVersion.targets(9,9): Warning MSB3884: Could not find rule set file "MinimumRecommendedRules.ruleset". (MSB3884) (Steam Desktop Authenticator)`
Help is really appreciated!

**Comment by dyc3** [comment]
run it with `mono`

`mono [file]`

**Comment by vasikxl** [comment]
Just misplaced code tag, edited, but thanks!
Or you mean like run it like this:
`mono "/home/.../SteamDesktopAuthenticator/Steam Desktop Authenticator/bin/release/Steam Desktop Authenticator.exe"`
?

**Comment by dyc3** [comment]
No, you are using `mono` correctly. If you are still having issues, you can try to use [steamguard-cli](https://github.com/dyc3/steamguard-cli)
